subprojects/emcee/android-device/src/main/kotlin/com/avito/emcee/device/AndroidDevice.kt

```diff
@@ -0,0 +1,9 @@
+package com.avito.emcee.device
+
+public interface AndroidDevice {
+    public suspend fun installApplication(application: AndroidApplication): Result<Unit>
```

Comment:
Should be mirrored with `uninstall` fun, i propose to remove `Application` from name here